src/build/arg/mod.rs

```diff
@@ -105,15 +104,20 @@ impl<'help> Arg<'help> {
         self.help.clone()
     }
 
-    /// Get the short option for this argument, if any
+    /// Get the short option name for this argument, if any
     pub fn get_short(&self) -> Option<char> {
         self.short.clone()
     }
 
-    /// Get the long option for this argument, if any
+    /// Get the long option name for this argument, if any
     pub fn get_long(&self) -> Option<&str> {
         self.long.clone()
     }
+
+    /// Get the list of the possible values for this argument, if any
+    pub fn get_possible_values(&self) -> Option<&[&str]> {
```

Comment:
Should we name this `get_possible_vals` since the data is `possible_vals`? What's the name of the builder for this data?

Reply:
[`possible_values`](https://docs.rs/clap/2.33.0/clap/struct.Arg.html#method.possible_values)